README.md

```diff
@@ -1,12 +1,29 @@
-# Aria-AT Report
-This is a web application for running manual test material from the [w3c/aria-at](https://github.com/w3c/aria-at) project, in order to evaluate the interoperability of screen reader interpretations of ARIA.
+# ARIA and Assistive Technologies App (ARIA-AT App)
+## ARIA-AT APP
+ARIA-AT aims to improve interoperability between different Assistive Technologies (ATs) in how they render ARIA patterns. This is achieved through running manual tests and presenting test results to AT vendors. The tests are based on examples from WAI-ARIA Authoring Practices, and are vetted with stakeholders following the working mode process.
+
+This app includes a reports page that shows the public results of manual testing across various browser / AT combinations. The app is also used by manual testers
+
+## About this Software
+This repo contains the react, node and graphql software for running the web application that manages testers to run the manual tests, launches the tests in an iframe, stores results, and reports results once they are reviewed.
```

Comment:
there is no graphql. But you might want to say database design via sequelize

Reply:
updated.